kroxylicious-api/src/main/java/io/kroxylicious/proxy/plugin/UnknownPluginInstanceException.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy.plugin;
+
+/**
+ * A reference to a plugin instance could not be resolved by the proxy runtime.
+ */
+public class UnknownPluginInstanceException extends RuntimeException {
```

Comment:
Does having two different exceptions really help? 

I wonder if it would make more sense to have a single `PluginLoadingException` and then distinguish the cases through the message?